src/variants.ini

```diff
@@ -575,6 +575,9 @@ promotionPawnTypes = -
 maxRank = 6
 maxFile = 6
 startFen = 2bnrk/5p/6/6/P5/KRNB2
+promotionPieceTypes = nbr
+promotionRegionWhite = *6
+doubleStep = f
```

Comment:
Thanks for your PR. Technically anything other than `true` works here, but for consistency it would be good to change to `false`.